espresso-server/app/src/test/java/io/appium/espressoserver/test/helpers/w3c/PointerDispatchTest.java

```diff
@@ -67,7 +68,7 @@ public void shouldNoopPointerMoveIfNoButtons() throws ExecutionException, Interr
     public void shouldDoOneMoveIfDurationZero()
             throws ExecutionException, InterruptedException, AppiumException {
         DummyW3CActionAdapter dummyW3CActionAdapter = new DummyW3CActionAdapter();
-        pointerInputSource = new PointerInputState();
+        pointerInputSource = new PointerInputState(TOUCH);
```

Comment:
this is not needed. Just add the default constructor, which assigns TOUCH type by default

Reply:
I'd rather it be explicit because the reason this bug happened in the first place was because I didn't make it mandatory to set the touch type.